tests/fixtures.py

```diff
@@ -25,6 +26,12 @@ def get_test_profile(self):
     def setUp(self):
         app.config['TESTING'] = True
         app.testing = True
+
+        # PYTHONPATH is `pwd` when testing and empty otherwise
+        # using it to determine the project root
+        # either get it from environment variable, or it's one level up from this init file
+        app.project_root = os.environ.get('PYTHONPATH', '') or os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
`PYTHONPATH` is a path and can potentially contain multiple semicolon-separated entries. Best to use the `dirname` method you have here and use that alone.